programs/token-2022/src/instructions/extensions/memo_transfer/state.rs

```diff
@@ -0,0 +1,25 @@
+#[repr(u8)]
```

Comment:
This file seems unnecessary. Just define the constants for `Enable` and `Disable` where they are used.